doc/nrf/app_dev/device_guides/fem/fem_simple_gpio.rst

```diff
@@ -67,6 +67,14 @@ To use the Simple GPIO implementation of FEM with SKY66112-11, complete the foll
    .. note::
        This step is not needed when testing with the :ref:`direct_test_mode` or :ref:`radio_test` samples on nRF53 Series devices.
 
+#. On nRF54L Series devices, make sure the GPIO pins of the SoC selected to control ``ctx-gpios`` and ``crx-gpios`` support GPIOTE.
+   For example, on the nRF54L15 device, use pins belonging to GPIO P1 or GPIO P0 only.
+   You cannot use the GPIO P2 pins, because there is no related GPIOTE peripheral.
+   It is recommended to use the GPIO pins that belong to the PERI Power Domain of the nRF54L device.
+   For example, on the nRF54L15, these are pins belonging to GPIO P1.
+   Using pins belonging to Low Power Domain (GPIO P0 on nRF54L15) is supported but requires more DPPI and PPIB channels of the SoC.
+   Ensure the instances ``dppic10``, ``dppic20``, ``dppic30``, ``ppib11``, ``ppib21``, ``ppib22``, ``ppib30`` are enabled (have ``status = "okay";``) in the devicetree.
```

Comment:
```suggestion
   Ensure that the following devicetree instances are enabled (have ``status = "okay"``):
   
   * ``dppic10``
   * ``dppic20``
   * ``dppic30``
   * ``ppib11``
   * ``ppib21``
   * ``ppib22``
   * ``ppib30`
```

Reply:
@peknis this is written in the same way for other FEMs. Could is stay as is?

Reply:
Okay for me. Just want to know whether the semicolon is needed after "okay". If not, you could take it out from other FEM docs as well.